common/components/filter/_filter.scss

```diff
@@ -0,0 +1,67 @@
+.app-filter {
+  margin: 0 0 govuk-spacing(3);
+}
+
+.app-filter__list {
+  @include govuk-clearfix;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.app-filter__list-item {
+  flex-grow: 1;
+  min-height: 70px;
+  display: flex;
+}
+
+.app-filter__list-item--active {
+  background: govuk-colour('white');
+  border: 2px solid govuk-colour('black');
+  padding: govuk-spacing(4);
+  padding-right: govuk-spacing(6);
+  @include govuk-font($size: 19, $weight: normal, $line-height: 1.25);
```

Comment:
Do we need to specify the line-height in this mixin? I think `1.25` and `normal` are the defaults.